ui/src/onboarding/containers/LoginPage.tsx

```diff
@@ -0,0 +1,50 @@
+import React, {FC} from 'react'
+
+import {
+  AlignItems,
+  AppWrapper,
+  Columns,
+  FlexBox,
+  FlexDirection,
+  Grid,
+  JustifyContent,
+  Page,
+  Panel,
+} from '@influxdata/clockface'
+
+// import LoginPage from 'src/onboarding/containers/LoginPage'
+
+const SignUp: FC = () => (
+  <AppWrapper className="sign-up--page">
+    <Page titleTag="Sign Up for InfluxDB Cloud">
+      <Page.Contents
+        scrollable={true}
+        fullWidth={true}
+        className="sign-up--page-contents"
+      >
+        <Panel className="sign-up--panel">
+          <FlexBox
+            direction={FlexDirection.Column}
+            stretchToFitHeight={true}
+            justifyContent={JustifyContent.Center}
+            alignItems={AlignItems.Center}
+          >
+            <Grid.Row className="sign-up--full-height">
+              <Grid.Column
+                widthXS={Columns.Twelve}
+                widthMD={Columns.Five}
+                offsetMD={Columns.Four}
+                widthLG={Columns.Four}
+                className="sign-up--full-height"
+              >
+                Login Page will go here
```

Comment:
is this text being in here a problem? i'm assuming no cause of flagging but it's good to ask

Reply:
Not an issue in compiling, but it's also just a temporary placeholder until the next PR